src/components/application_manager/test/commands/mobile/put_file_test.cc

```diff
@@ -345,18 +345,22 @@ TEST_F(PutFileRequestTest, Run_InvalidPutFile_UNSUCCESS) {
   command->Run();
 }
 
-TEST_F(PutFileRequestTest, Run_SrcSumEqual_SendSuccessResponse) {
-  (*msg_)[am::strings::msg_params][am::strings::crc32_check_sum] = 2768625435u;
+TEST_F(PutFileRequestTest, Run_CrcSumEqual_SendSuccessResponse) {
+  binary_data_ = {1u};
+  (*msg_)[am::strings::params][am::strings::binary_data] = binary_data_;
+  uint32_t correct_crc_sum = 2768625435u;
```

Comment:
@BSolonenko const and add comment

Reply:
@AByzhynar Fixed in https://github.com/smartdevicelink/sdl_core/pull/2139/commits/b05aee0fa7da035c66c5c76041339efd57ee511c